src/tracer.js

```diff
@@ -34,6 +34,13 @@ export default class Tracer {
      *         childOf: parent.context(),
      *     });
      *
+     *     // Start a new async (FollowsFrom) Span:
+     *     var child = Tracer.startSpan('Subroutine', {
```

Comment:
nit: `s/'Subroutine'/'pipeConsumer'` (or something else that implies `followsFrom`)